### PR TITLE
Add eager_pip to simple_console_for_windows.zip

### PR DESCRIPTION
**Pull request by snnn**
To fix unitest errors in tf-master-win-bzl build

http://ci.tensorflow.org/job/tf-master-win-bzl/1491/console

### CODE REVIEW

**Review by asimshankar — APPROVED** [review]
Thanks for the fix, much appreciated.

**Comment by asimshankar** [comment]
Jenkins, test this please

**Comment by meteorcloudy** [comment]
Testing this change on Windows: http://ci.tensorflow.org/job/tensorflow-pr-win-bazel/37/console

**Comment by snnn** [comment]
Hi @meteorcloudy  Could you please re-try it? Thanks.

**Comment by meteorcloudy** [comment]
Retrying here: http://ci.tensorflow.org/job/tensorflow-pr-win-bazel/38/console
Only three tests are broken now:
```
01:31:18 //py_test_dir/tensorflow/python/eager:backprop_test                      FAILED in 29.6s
01:31:18   C:/tmp/_bazel_system/7rgg5n4w/execroot/org_tensorflow/bazel-out/msvc_x64-py3-opt/testlogs/py_test_dir/tensorflow/python/eager/backprop_test/test.log
01:31:18 //py_test_dir/tensorflow/python/kernel_tests:checkpoint_ops_test         FAILED in 18.8s
01:31:18   C:/tmp/_bazel_system/7rgg5n4w/execroot/org_tensorflow/bazel-out/msvc_x64-py3-opt/testlogs/py_test_dir/tensorflow/python/kernel_tests/checkpoint_ops_test/test.log
01:31:18 //py_test_dir/tensorflow/python:saver_test                               FAILED in 42.6s
01:31:18   C:/tmp/_bazel_system/7rgg5n4w/execroot/org_tensorflow/bazel-out/msvc_x64-py3-opt/testlogs/py_test_dir/tensorflow/python/saver_test/test.log
```

**Comment by caisq** [comment]
ping @snnn

**Comment by snnn** [comment]
sorry, I'm trying to find out why backprop_test  was failed. Any suggestions? 

Indeed, I have another pull request #12296 ,  which **almost** contains all the changes of this. So, we can abandon this if that was approved.

**Comment by meteorcloudy** [comment]
@asimshankar, since you added backprop_test, any idea why it's failing on Windows with:
```
01:30:24 FAIL: //py_test_dir/tensorflow/python/eager:backprop_test (see C:/tmp/_bazel_system/7rgg5n4w/execroot/org_tensorflow/bazel-out/msvc_x64-py3-opt/testlogs/py_test_dir/tensorflow/python/eager/backprop_test/test.log)
01:30:24 INFO: From Testing //py_test_dir/tensorflow/python/eager:backprop_test:
01:30:24 ==================== Test output for //py_test_dir/tensorflow/python/eager:backprop_test:
01:30:24 C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\util\tf_inspect.py:45: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() instead
01:30:24   if d.decorator_argspec is not None), _inspect.getargspec(target))
01:30:24 ...ss.........Ess.
01:30:24 ======================================================================
01:30:24 ERROR: testSecondGrad (__main__.BackpropTest)
01:30:24 ----------------------------------------------------------------------
01:30:24 Traceback (most recent call last):
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 251, in vspace
01:30:24     return vspace_mappings[type(value)](value)
01:30:24 KeyError: <class 'tensorflow.python.eager.tensor_node.TensorNode'>
01:30:24 
01:30:24 During handling of the above exception, another exception occurred:
01:30:24 
01:30:24 Traceback (most recent call last):
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\errors.py", line 48, in wrapped
01:30:24     try: return fun(*args, **kwargs)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\convenience_wrappers.py", line 23, in gradfun
01:30:24     vjp, ans = make_vjp(fun, argnum)(*args, **kwargs)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 12, in vjp_maker
01:30:24     start_node, end_node = forward_pass(fun, args, kwargs, argnum)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 23, in forward_pass
01:30:24     start_node = new_progenitor(args[argnum])
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 145, in new_progenitor
01:30:24     node = new_node(x,       (identity, (x,), {}, []      ), set())
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 245, in new_node
01:30:24     return node_type_mappings[type(value)](value, recipe, progenitors)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 160, in __init__
01:30:24     self.vspace = vspace(value)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 251, in vspace
01:30:24     return vspace_mappings[type(value)](value)
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\container_types.py", line 66, in __init__
01:30:24     self.shape = [vspace(x) for x in value]
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\container_types.py", line 66, in <listcomp>
01:30:24     self.shape = [vspace(x) for x in value]
01:30:24   File "C:\Program Files\Anaconda3\lib\site-packages\autograd\core.py", line 253, in vspace
01:30:24     raise TypeError("Can't find vspace for type {}".format(type(value)))
01:30:24 TypeError: Can't find vspace for type <class 'tensorflow.python.eager.tensor_node.TensorNode'>
```

**Comment by asimshankar** [comment]
Hmm...that is most likely because the version of the `autograd` pip package installed on the test machine is old. I'm not familiar enough with how the Jenkins Windows machines get set up, perhaps we can chat off-thread on how to (a) update the version, and (b) ensure that updates to the required version happen automatically

(FYI @alextp )

**Comment by caisq** [comment]
This might be due to an old version of autograd installed on the Jenkins Windows slaves. The testing-infra folks can log in and find out whether that's the case. We use autograd==1.1.12 on Linux and Mac.

cc @gunan, @av8ramit, @yifeif

File: tensorflow/tools/pip_package/BUILD [hunk 1]
```diff
@@ -84,6 +84,7 @@ py_binary(
         "//tensorflow/python/saved_model",
         "//tensorflow/python:spectral_ops_test_util",
         "//tensorflow/python/tools:tools_pip",
+        "//tensorflow/python/eager:eager_pip",
```
Comment (caisq) [inline tensorflow/tools/pip_package/BUILD]: nit: @snnn We aim to keep this list alphabetically sorted. Can you fix the ordering?

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
Upgrading autograd from 1.1.11 to 1.1.12 on the Windows Jenkins resolved the issue. Merging PR now. @snnn, thanks for the contribution! We can fix the ordering later.

**Pull request merged**